src/engraving/dom/utils.cpp

```diff
@@ -1183,6 +1183,9 @@ SymIdList timeSigSymIdsFromString(const String& string, TimeSigStyle timeSigStyl
         { 162,   SymId::timeSigCutCommon },             // '¢'
         { 189,   SymId::timeSigFractionHalf },
         { 188,   SymId::timeSigFractionQuarter },
+        { 88,    SymId::timeSigMultiply },              // 'X'
+        { 120,   SymId::timeSigMultiply },              // 'x'
+        { 215,   SymId::timeSigMultiply },              // '×'
```

Comment:
Any special reason why not using the characters themselves rather than their ASCII code?
As far as I can tell `Char(88)` is the same as `Char('X')`

Reply:
@rettinghaus I agree with Jojo's comment here ^